packages/svelte/src/internal/client/render.js

```diff
@@ -2843,6 +2843,8 @@ export function spread_attributes(dom, prev, attrs, css_hash) {
 	if (!setters) map_set(setters_cache, dom.nodeName, (setters = get_setters(dom)));
 
 	for (const key in next) {
+		if (key.startsWith('$$')) continue;
```

Comment:
Can we get the first two characters of the key? Then we can compare that using `===` below for here and `on`, which will be far faster then doing `startsWith` twice (I think).

Reply:
yep, tweaked it a bit